tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py

```diff
@@ -84,6 +92,11 @@ def a11y_toggle_switch(self, switch):
     def a11y_toggle_filters(self):
         self.a11y_toggle_switch(self.marionette.find_element(*self._filter_enable_switch_locator))
 
+    def toggle_filters(self):  # non-a11y way of toggling the switch
+        Wait(self.marionette).until(
+            expected.element_displayed(*self._filter_enable_switch_locator))
+        self.marionette.find_element(*self._filter_enable_switch_locator).tap()
```

Comment:
Since you use need the element twice here, it would have been better to do something like this:
element = self.marionette.find_element(*self._filter_enable_switch_locator)
Wait(self.marionette).until( expected.element_displayed(element))
element.tap()
That way, marionette has to find that element only once.